tools-local/tasks/mobile.tasks/WasmBuildTasks/CompileTimeZoneData.cs

```diff
@@ -49,7 +51,8 @@ private void FilterTimeZoneData()
         //  for ex: `CST6CDT`, `MST`, etc.
         foreach (var entry in new DirectoryInfo (OutputDirectory!).EnumerateFiles())
         {
-            File.Delete(entry.FullName);
+            if (entry.Name != "zone.tab")
```

Comment:
Why is this needed? We never copy this file to the `OutputDirectory`, so it shouldn't be there at all, IIUC.

Reply:
It is copied to the output directory (read above).

Reply:
AFAIU, earlier behavior was:

1. run `zic` # this does not create the zone.tab file
2. `FilterTimeZoneData` deletes bunch of files like for `MST` (does *not* exclude `zone.tab`, but it isn't there, so it doesn't matter)
3. `FilterZoneData` - creates a new `zone.tab` by filtering `zone1970.tab`

Now:

1. run `zic` # this does not create the zone.tab file
2. `FilterZoneData` - creates a new `zone.tab` by filtering `zone1970.tab`
3. `FilterTimeZoneData` deletes bunch of files like for `MST`

So, step 3 creates a `zone.tab`. And because the patch moves step 3 => 2, the condition for skipping `zone.tab` is required. Instead, you can retain the order as it was earlier, and no check is required.

Reply:
I added some logging to the task, which should be useful for debugging, and should make the output cleaner.
https://gist.github.com/radical/00989f366f3f4bc263f06d87dd36c4e1

Reply:
I looked it over and it looks good to me. Feel free to push to my branch.